addons/web/static/src/core/datepicker/datepicker.js

```diff
@@ -171,15 +171,15 @@ export class DatePicker extends Component {
 
     /**
```

Comment:
Can you be rewrite the second phrase and split it in several? You should at least mention the datepicker